### PR TITLE
Added missing test results to stdout

### PR DESCRIPTION
**Pull request by Xenwar**

### CODE REVIEW

File: upgrade_tests/controlplane_upgrade/3cp_0w_bootDiskImage_extraNode_upgrade.sh [hunk 1]
```diff
@@ -92,6 +92,7 @@ for i in {1..3600};do
 		  exit 1
   fi
 done
+echo "Upgrading of boot disk image of all controlplane nodes with extra node has succeeded"
```
Comment (Jaakko-Os) [inline upgrade_tests/controlplane_upgrade/3cp_0w_bootDiskImage_extraNode_upgrade.sh:95]: 'disk image'
I think
'node image' is the one used

File: upgrade_tests/worker.division.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
+Anwar
```
Comment (Jaakko-Os) [inline upgrade_tests/worker.division.sh:1]: don't push this file

Comment (Xenwar) [reply upgrade_tests/worker.division.sh]: fixed

File: upgrade_tests/controlplane_upgrade/3cp_0w_bootDiskImage_extraNode_upgrade.sh (hunk 1 above)
Comment (Xenwar) [reply upgrade_tests/controlplane_upgrade/3cp_0w_bootDiskImage_extraNode_upgrade.sh]: fixed

**Pull request merged**